{"name": "test_user_login_with_incorrect_data", "status": "passed", "description": "Логин с неверным логином и паролем", "start": 1724060686784, "stop": 1724060687514, "uuid": "6e8e17a3-4775-46de-a328-840c21dc9029", "historyId": "88e9223a586f217712e0f1194e40759e", "testCaseId": "88e9223a586f217712e0f1194e40759e", "fullName": "tests.test_user_login#test_user_login_with_incorrect_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "host", "value": "DESKTOP-0A2OPDH"}, {"name": "thread", "value": "8544-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}